frontend/src/pages/DashboardPage.tsx

```
@@ -0,0 +1,67 @@
+
+  return (
+    <div className="min-h-screen bg-gray-50">
+      <nav className="border-b bg-white shadow-sm">
+        <div className="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
+          <div className="flex h-16 justify-between">
+            <div className="flex items-center">
+              <h1 className="text-xl font-bold text-gray-900">Safe Job Dashboard</h1>
+            </div>
+            <div className="flex items-center space-x-4">
+              <span className="text-sm text-gray-600">Welcome, {user?.firstName}</span>
+              <button onClick={logout} className="btn-secondary">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)